Fix yarn dev script

cdce8p opened the pull request.

cdce8p committed:
Fix yarn dev script

balloob commented on package.json:
```diff
@@ -9,7 +9,7 @@
   "scripts": {
     "clean": "rm -rf build/* build-temp/* build-es5/* build-temp-es5/* build-translations/*",
     "build": "script/build_frontend",
-    "dev": "npm run gulp ru_all gen-service-worker",
```
This should just run `script/develop` or even better, just be removed.

cdce8p committed:
Removed yarn dev

cdce8p commented:
I removed it. However `./node_modules/.bin/gulp gen-service-worker` needs to be executed at least once if I'm not mistaken. The frontend didn't work without it for me. Therefore I added it to `script/bootstrap`.

cdce8p commented:
I also opened a PR for the `dev` docs (for `script/develop`): https://github.com/home-assistant/developers.home-assistant/pull/15

balloob commented on script/bootstrap:
```diff
@@ -16,4 +16,4 @@ yarn install
 
 # Build files that need to be generated to run development mode
-yarn dev
+./node_modules/.bin/gulp gen-service-worker
```
Bootstrap should just run yarn. All other lines can go in favor of `script/devel`

balloob committed:
Update bootstrap
